test/ui-test/testSuites/global_shared/steps/steps.py

```diff
@@ -114,8 +106,3 @@ def the_user_inputs_the_following_text_with_uicomponent(text: str, obj):
 def the_user_clicks_on_the_following_ui_component(obj):
     init_steps.the_user_clicks_on_the_following_ui_component(obj)
 
-def the_user_joins_chats(context: any):
-    table = context.table
-    for i, row in enumerate(table):
-        room = row[0]
-        init_steps.the_user_joins_chat_room(room)
```

Comment:
Judging by the PR/commit it's not obvious why we're removing these here.
Isn't this PR just supposed to re-add stuff?

I don't know having those changes here as well, but maybe we should have at least a comment here on why this is happening

Reply:
Ah sorry, it's code that should have been removed when I removed the public chat and public chats test in my other PR, but I now just realized it while working on it.

Sorry for the confusion.